src/components/input/input-theme.scss

```diff
@@ -37,6 +37,66 @@ md-input-container.md-THEME_NAME-theme {
         color: '{{foreground-2}}';
       }
     }
+  }
+  .md-input {
+    &[disabled],
+    [disabled] & {
+      border-bottom-color: transparent;
+      color: '{{foreground-3}}';
+      background-image: linear-gradient(to right, '{{foreground-3}}' 0%, '{{foreground-3}}' 33%, transparent 0%);
+      background-image: -ms-linear-gradient(left, transparent 0%, '{{foreground-3}}' 100%);
+    }
+  }
+}
+
+md-toolbar.md-THEME_NAME-theme:not(.md-menu-toolbar) md-input-container {
```

Comment:
Yes, the reason it's in input-theme.scss because overrides. Of the three SCSS collections, the third needs to override the second and the third also shares values.

Quick ref:

Second Collection:

    md-toolbar.md-THEME_NAME-theme:not(.md-menu-toolbar) md-input-container {

Third Collection:

    md-toolbar.md-THEME_NAME-theme:not(.md-menu-toolbar) md-input-container,
    md-input-container.md-THEME_NAME-theme

It's a weird order of overrides, but I feel like perhaps there should be a `@mixin` where the backcolor could change and include md-toolbar in there. In fact, off the top of my head, I can see colored `md-card`s with child md-input-containers having the same issue as when it was in a toolbar.